docs-website/sidebars.js

```diff
@@ -106,6 +106,15 @@ module.exports = {
         ],
       },
     ],
+    "Integration Guides": [
```

Comment:
@laulpogan curious for your input here - where do you want these to go?

Looking at it on my local, my suggestion would be this layout: 
![CleanShot 2022-12-08 at 21 07 51](https://user-images.githubusercontent.com/15873986/206615474-f59b08bb-a7e2-4595-aee4-41b35205cfde.png)

Where do you see these living in the long run?

Reply:
These are either getting started or ingestion guides, I like them better in the quickstart area, keeping in mind that we are trying to segregate reference (Sources here) into its own category in the future. Right now I think that we should keep it out of that section to avoid confusing people and having them ask where other guides are